proto/one/operations/sample/test_analyte_group.proto

```diff
@@ -25,7 +26,10 @@ message TestAnalyteGroup {
     google.protobuf.StringValue notes = 6;
 
     // Soft Delete of the test analyte group
-    bool isActive = 7;   
+    bool isActive = 7;
+
+    // Collection of analytes in the test analyte group
+    repeated Analyte Analytes = 8;
```

Comment:
should the property name be "analytes" instead of "Analytes"?

Reply:
Yes, Good catch. I pushed the fix.